gcloud-java-compute/src/main/java/com/google/gcloud/compute/testing/RemoteComputeHelper.java

```diff
@@ -52,6 +53,14 @@ public ComputeOptions options() {
   }
 
   /**
+   * Returns a base name for testing resources generated using a random UUID. This base name can be
+   * prepended to resource names to prevent name clashes.
```

Comment:
Should we mention it in the java doc and suggests that this prefix size is going to be 30 therefore their suffix shouldn't exceed 33.